charts/greeting/templates/01-configmap_greeting-props.yaml

```diff
@@ -1,8 +1,9 @@
 apiVersion: v1
 data:
-  application.properties: |2
-
-    quarkus.flyway.migrate-at-start=true
+  application.properties: "# This property is used to select the log level, which
```

Comment:
@ElaiShalevRH  @gabriel-farache any idea how to maintain a proper rendering of this section?